extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/stream/YoutubeStreamExtractorLivestreamTest.java

```diff
@@ -57,9 +57,9 @@ public void testUploaderName() throws Exception {
     @Override public long expectedLength() { return 0; }
     @Override public long expectedTimestamp() { return TIMESTAMP; }
     @Override public long expectedViewCountAtLeast() { return 0; }
-    @Nullable @Override public String expectedUploadDate() { return "2020-02-22 00:00:00.000"; }
-    @Nullable @Override public String expectedTextualUploadDate() { return "2020-02-22"; }
-    @Override public long expectedLikeCountAtLeast() { return 825000; }
```

Comment:
They've replaced the stream that was referenced by the test previously. Tests against live streams seem rather fragile -- do we have a better approach to this? Just testing against mocks?
Would be good to still test against the live API.. so perhaps, just find the first stream and check that we can extract non-empty values?